src/app/patches/patch6/patch6.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="container">
+    <div class="row">
+        <div class="col pl-0 pr-0">
+            <nav class="navbar navbar-expand-lg navbar-dark" id="navbarMenu">
+                <ul class="navbar-nav mr-auto">
+                    <li class="nav-item">
+                        <h5 class="active nav-link" id="head">Patch6</h5>
+                    </li>
+                </ul>
+                <ul class="nav navbar-nav navbar-right">
+                    <li class="nav-item">
+                        <span type="button" class="material-icons text-light" (click)="backToDashboard()">close</span>
+                    </li>
+                </ul>
+            </nav>
+        </div>
+    </div>
+    <div class="px-6" id="weigh">
+        <div class="container p-3 mr-3 my-3">
+            <div class="row mb-2">
+                <div class="col-md-6">
+                    <div>Creation Date: <span class="pl-3">2/10/2021</span></div>
+                </div>
+                <div class="col-md-6">
+                    <div class="pr-3">Last Updated On: <span class="pl-3">2/10/2021</span></div>
```

Comment:
Update the right date

Reply:
Create a feature even this data should be fetched automatically from db.